### PR TITLE
Feature/Add toggle to disable automatic identiy creation

### PR DESCRIPTION
**Pull request by sebbi08**

### CODE REVIEW

File: packages/app-runtime/src/AppConfig.ts [hunk 1]
```diff
@@ -24,7 +24,8 @@ export function createAppConfig(...configs: AppConfigOverwrite[]): AppConfig {
         accountsDbName: "accounts",
         transportLibrary: {
             realm: Realm.Prod,
-            datawalletEnabled: true
+            datawalletEnabled: true,
+            allowIdentityCreation: true
```
[the remark this answers is not in the document]
Comment (jkoenig134) [reply packages/app-runtime/src/AppConfig.ts:28]: yep, I think so.

File: packages/transport/src/core/Transport.ts [hunk 2]
```diff
@@ -56,6 +58,7 @@ export class Transport {
     }
 
     private static readonly defaultConfig: IConfig = {
+        allowIdentityCreation: false,
```
Comment (jkoenig134) [inline packages/transport/src/core/Transport.ts:61]: I would vote for leaving the functionality as-is by using true as a default and using false in the connector

File: packages/transport/src/modules/accounts/AccountController.ts
Comment (jkoenig134) [inline packages/transport/src/modules/accounts/AccountController.ts:1]: I'm a bit confused that your vscode sorts imports completely different than mine. Is it up2date?

Comment (sebbi08) [reply packages/transport/src/modules/accounts/AccountController.ts:1]: Yea also took a look into it, do you have some extension installed that alters the sorting ? 

Maybe we should add [this](https://eslint.org/docs/latest/rules/sort-imports) to our eslint config

Comment (jkoenig134) [reply packages/transport/src/modules/accounts/AccountController.ts:1]: No, I use vscode default behaviour.

Comment (sebbi08) [reply packages/transport/src/modules/accounts/AccountController.ts:1]: Hm weird, when I resave the file the imports go back to the right order. Like I had not saved the file before ^^

**Comment by jkoenig134** [comment]
don't you want to publish all this?

**Comment by jkoenig134** [comment]
version bump not added to lockfile.... `npm i` please

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**